db/init_db.js

```diff
@@ -459,6 +453,36 @@ async function createInitialUsers() {
     throw error;
   };
 };
+async function createInitialOrders() {
+  console.log("Starting to create orders");
+  try {
+    const ordersData = [
+      {status: "created", userId: "1"},
+      {status: "completed", userId: "2"},
+     ]
+
+    await Promise.all(ordersData.map(createOrder));
+
+  } catch (error) {
+    throw error;
+  };
+};
+
+//waiting for db 
+async function createInitialOrderProducts() {
+  console.log("Starting to create order_products");
+  try {
+    const orderProductsData = [
+      {productId: "1", orderId: "1", price: "88", quantity: "1", userId:"1" },
+      {productId: "2", orderId: "2", price: "35", quantity: "2", userId:"2" },
+     ]
+
+    await Promise.all(orderProductsData.map(getOrdersByProduct));
```

Comment:
I'll wait for @michellemal's PR before I merge this one, so that we have the correct function for creating initial order products.

Reply:
updated and resolved.